src/amber/cli/templates/app/src/assets/javascripts/main.js

```diff
@@ -1 +1,23 @@
 import Amber from 'amber'
+
+if (!Date.prototype.toGranite) {
+  (function() {
+
+    function pad(number) {
+      if (number < 10) {
+        return '0' + number;
+      }
+      return number;
+    }
+
+    Date.prototype.toGranite = function() {
+      return this.getUTCFullYear() +
+        '-' + pad(this.getUTCMonth() + 1) +
+        '-' + pad(this.getUTCDate()) +
+        ' ' + pad(this.getUTCHours()) +
+        ':' + pad(this.getUTCMinutes()) +
+        ':' + pad(this.getUTCSeconds())  ;
+    };
```

Comment:
Does javascript not have a strfmt? I'm surprised javascript doesn't have an iso-8601 formatter that's compatible with granite.

Reply:
https://www.npmjs.com/browse/keyword/strftime

Which one will not be another `left_pad`?

Reply:
[how about this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)

Reply:
Javascript Date has toISOString() but it generates a string with a T between the date and time.  I often use moment library to format dates in the presentation layer but of course I didn't want to introduce an unnecessary dependency when a simple function can do the job.